### ISSUE TITLE
Duplicated exception console output in 2.x

### ISSUE THREAD

**Issue by davidmatson** [Azure/azure-webjobs-sdk#2188]
I'd love to upgrade to 3.x but cannot due to #2186.

In the meantime, can the following change be merged back into 2.x?
https://github.com/Azure/azure-webjobs-sdk/commit/d88f6e1467d598d23fd8ea13e67aad7082d82a63#diff-7609f499d07d276aa8d51402641edd3c

Thanks!

**Comment by brettsam** [comment]
Agree. It likely won't be the same fix (if I recall that other one had to do with the Functions CLI and some DI stuff) but I'm seeing dupe errors in a simple repro.
